core/src/components/input/input.ionic.outline.scss

```diff
@@ -0,0 +1,110 @@
+@import "./input.vars";
+
+// Input Fill: Outline (Ionic Theme)
+// ----------------------------------------------------------------
+
+:host(.input-fill-outline) {
+  --border-radius: 4px;
+  --padding-start: 12px;
+  --padding-end: 12px;
+  --placeholder-color: #535353;
+  --placeholder-opacity: 1;
+}
+
+/**
+ * The bottom content should never have
+ * a border with the outline style.
+ */
+:host(.input-fill-outline) .input-bottom {
+  border-top: none;
+}
+
+:host(.input-fill-outline) .input-wrapper {
+  /**
+   * For the ionic theme, the padding needs to sit on the
+   * native wrapper instead, so that it sits within the
+   * outline container but does not affect the label text.
+
+   * For the ionic theme, the horizontal padding needs to
+   * sit on the native wrapper instead, so that
+   */
+  @include padding(0);
+
+  /**
+   * Outline inputs do not have a bottom border.
+   * Instead, they have a border that wraps the
+   * input + label.
+   */
+  border-bottom: none;
+}
+
+:host(.input-fill-outline.input-label-placement-stacked) .label-text-wrapper {
+  @include transform-origin(start, top);
+
+  /**
+   * Label text should not extend
+   * beyond the bounds of the input.
+   */
+  max-width: calc(100% - var(--padding-start) - var(--padding-end));
+}
+
+:host(.input-fill-outline) .label-text-wrapper {
+  /**
+   * The label should appear on top of an outline
+   * container that overlaps it so it is always clickable.
+   */
+  position: relative;
+
+  color: #373737;
+}
+
+:host(.input-fill-outline) .native-wrapper {
+  @include padding(var(--padding-top), var(--padding-end), var(--padding-bottom), var(--padding-start));
+
+  min-height: 40px;
```

Comment:
This change causes the `large` [size styling](https://github.com/ionic-team/ionic-framework/blob/41df86257753ba45da5f8c2ce02f795807ee7726/core/src/components/input/input.ionic.scss#L19-L21) to no longer work. We'll need to adjust the large size for the outline.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/29268/commits/9b482ae61602c71129b9826bbc3ead86953d3419, screenshots updated in https://github.com/ionic-team/ionic-framework/pull/29268/commits/b39e5bc75946b693c2be0db7ea32dc5f4b268b1f